Error: Cannot find module 'axios'

<!--- Provide a general summary of the issue in the Title above -->

When including the plugin as a testing dependency into a Cypress project, it causes Cypress to fail when running any tests due to a failed `axios` import.

## Expected Behavior

<!--- Tell us what should happen -->

The plugin should not result in an import error on Cypress test run setup.

## Current Behavior

<!--- Tell us what happens instead of the expected behavior -->

Currently, when running either `npx cypress run` or `npx cypress open`, the following error appears:

```ts
$ npx cypress run

Your configFile threw an error from: C:\repositories\some-repo\cypress.config.ts

The error was thrown while executing your e2e.setupNodeEvents() function:

Error: Cannot find module 'axios'
Require stack:
- C:\repositories\some-repo\node_modules\cypress-xray-plugin\src\credentials.js
- C:\repositories\some-repo\node_modules\cypress-xray-plugin\src\util\config.js
- C:\repositories\some-repo\node_modules\cypress-xray-plugin\src\hooks.js
- C:\repositories\some-repo\node_modules\cypress-xray-plugin\plugin.js
- C:\repositories\some-repo\cypress.config.ts
- C:\Users\some-user\AppData\Local\Cypress\Cache\12.3.0\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_require_async_child.js
- C:\Users\some-user\AppData\Local\Cypress\Cache\12.3.0\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\require_async_child.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Function.Module._resolveFilename (C:\Users\some-user\AppData\Local\Cypress\Cache\12.3.0\Cypress\resources\app\node_modules\tsconfig-paths\lib\register.js:75:40)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (C:\Users\some-user\AppData\Local\Cypress\Cache\12.3.0\Cypress\resources\app\node_modules\@cspotcode\source-map-support\source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (C:\repositories\some-repo\node_modules\cypress-xray-plugin\src\credentials.js:7:33)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.require.extensions.<computed> [as .js] (C:\Users\some-user\AppData\Local\Cypress\Cache\12.3.0\Cypress\resources\app\node_modules\ts-node\dist\index.js:851:20)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (C:\repositories\some-repo\node_modules\cypress-xray-plugin\src\util\config.js:6:23)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.require.extensions.<computed> [as .js] (C:\Users\some-user\AppData\Local\Cypress\Cache\12.3.0\Cypress\resources\app\node_modules\ts-node\dist\index.js:851:20)
    at Module.load (node:internal/modules/cjs/loader:1037:32)
    at Function.Module._load (node:internal/modules/cjs/loader:878:12)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)

```

## Steps to Reproduce

<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->

1. create an empty cypress project:

    1. `npm init` (and follow through all the steps, doesn't matter)
    2. `npm i -D cypress`

2. install the plugin as a dependency:
    1. `npm i -D cypress-xray-plugin`
3. run cypress:
    1. `npx cypress run`

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "cypress-xray-plugin",
-    "version": "1.0.3",
+    "version": "1.0.4",
     "description": "A Cypress plugin for uploading test results to Jira Xray",
     "main": "src/uploader.js",
     "types": "src/uploader.d.ts",
@@ -32,15 +32,17 @@
         "test": "mocha -r ts-node/register 'test/**/*.ts'",
         "build": "tsc"
     },
+    "dependencies": {
+        "axios": "^1.2.3",
+        "form-data": "^4.0.0"
+    },
     "devDependencies": {
         "@types/chai": "^4.3.4",
         "@types/chai-as-promised": "^7.1.5",
         "@types/mocha": "^10.0.0",
-        "axios": "^1.2.2",
         "chai": "^4.3.7",
         "chai-as-promised": "^7.1.1",
         "cypress": "^11.1.0",
-        "form-data": "^4.0.0",
         "mocha": "^10.1.0",
         "shx": "^0.3.4",
         "ts-node": "^10.9.1",

